webservice/soap/classes/class.ilNusoapUserAdministrationAdapter.php

```diff
@@ -148,6 +148,16 @@ function __registerMethods()
 								SERVICE_USE,
 								'ILIAS login function via LDAP');
 
+		// loginStudipUser()
+		$this->server->register('loginStudipUser',
+								array('sid' => 'xsd:string',
+									  'user_id' => 'xsd:int'),
+								array('sid' => 'xsd:string'),
+								SERVICE_NAMESPACE,
+								SERVICE_NAMESPACE.'#loginStudipUser',
+								SERVICE_STYLE,
+								SERVICE_USE,
+								'ILIAS login function for Stud.IP-Connection');
```

Comment:
According to the ILIAS Jour Fixe: Please mark this method/service as @deprecated and add a short note, because this service will be removed with ILIAS 5.3.x

- http://www.ilias.de/mantis/view.php?id=18992